Sources/TIM/Models/TIMOpenIDConfiguration.swift

```diff
@@ -14,12 +14,16 @@ public struct TIMOpenIDConfiguration {
 
     /// Scopes
     public let scopes: [String]
+    
+    /// Scopes
+    public let additionalParameters: [String, String]?
```

Comment:
Please make it non-optional and then default to a default empty-value

Reply:
Done